pkg/detectors/apacta/apacta_test.go

```diff
@@ -1,120 +1,113 @@
-//go:build detectors
-// +build detectors
-
 package apacta
 
 import (
 	"context"
 	"fmt"
 	"testing"
-	"time"
 
-	"github.com/kylelemons/godebug/pretty"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/google/go-cmp/cmp"
 
-	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
-	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/engine/ahocorasick"
 )
 
-func TestApacta_FromChunk(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
-	defer cancel()
-	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors2")
-	if err != nil {
-		t.Fatalf("could not get test secrets from GCP: %s", err)
-	}
-	secret := testSecrets.MustGetField("APACTA")
-	inactiveSecret := testSecrets.MustGetField("APACTA_INACTIVE")
+var (
+	validPattern   = "abcd1234-ef56-gh78-ij90-klmn1234opqr"
+	complexPattern = `
+	func main() {
+		url := "https://api.example.com/v1/resource"
+
+		// Create a new request with the secret as a header
+		req, err := http.NewRequest("POST", url, bytes.NewBuffer([]byte("{}")))
+		if err != nil {
+			fmt.Println("Error creating request:", err)
+			return
+		}
+		
+		apactaSecret := "Bearer abcd1234-ef56-gh78-ij90-klmn1234opqr"
+		req.Header.Set("Authorization", apactaSecret)
```

Comment:
**Feedback needed:** How does this test example look? I aimed to create something that reflects a real-world scenario. If it looks good, I can use similar examples in the other test cases.